sdk/daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/crypto/NodeHashSpec.scala

```diff
@@ -0,0 +1,1014 @@
+// Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
minor: name this file `NodeHashV1Spec` to make it clear which version of your hashing scheme it tests?